lib/src/fwupd_client.dart

```diff
@@ -118,11 +118,68 @@ enum FwupdInstallFlag {
   ignorePower,
 }
 
-class FwupdException implements Exception {}
+enum FwupdError {
```

Comment:
Why did you change from separate exception classes to using an enum? I previously did separate classes to make it easier for the API user to catch specific exceptions they want to target. Any trends in other Dart code that indicate we should do it this way?

Reply:
No strong opinions. The enum was directly copied from libfwupd. :)